hrpccharter.md

```diff
@@ -16,15 +16,15 @@ How are human rights and public interest policy considered in the development of
 
 The Human Rights and Policy Considerations Research Group is chartered to research of protocol development that is responsible towards and mindful of the human rights of others and whether standards and protocols can enable, strengthen or threaten human rights, as defined in the Universal Declaration of Human Rights (UDHR) [1] and the International Covenant on Civil and Political Rights (ICCPR) [2].
 
-Furthermore HRPC researches how protocols can influence policy concerns that intersect with human rights, and vice versa. This research group is a discursive resource for the community to ensure the development process fully recognizes these potential public policy impacts, addresses those impacts adequately, and builds evidence and guidance for policy makers on technical solutions and the necessary design tradeoffs that should be made. The Global Data Protection Regulation [3], principles of necessity and proportionality of surveillance [4], are examples of policy developments that have led to rich areas of work for the IETF through the Privacy Enhancements and Assessments research group (PEARG) and more such regulatory actions are expected as the digital age progresses.
+Furthermore HRPC researches how protocols can influence policy concerns that intersect with human rights, and vice versa. This research group is a discursive resource for the community to support the development process in recognizing these potential public policy impacts as well assist to address those impacts adequately. The Global Data Protection Regulation [3], principles of necessity and proportionality of surveillance [4], are examples of policy developments that have led to rich areas of work for the IETF through the Privacy Enhancements and Assessments research group (PEARG) and more such regulatory actions are expected as the digital age progresses.
```

Comment:
I'm proposing to remove this middle part here, as it is actually not fully clear to my what this means or what the intention is. However, I don't think this group should aim to directly provide guidance to policy makers (as it can't speak for the IETF). I'm sure there will be discussions in the group than then might be channeled by participants to policy makers which is welcome and can have a positive impact. But I would like to avoid to create expectation about the role of this group that it can't fulfil (aka speaking for the IETF).

Reply:
Thanks for the suggestion, Mirja.

I think the first two textual replacements just water down the intention, which is a kidn of nuance that doesn't really help in the chartering process-- we are either doing it or we aren't but we can't police the enthusiasm with which work is taken.

On whether HRPC "builds evidence and guidance for policy makers on technical solutions and the necessary design tradeoffs that should be made" I think we are better off with this area of work than if we are not. On the one hand, HRPC outputs will be read by policy makers and so I think we shouldn't pretend that they won't be. And on the other hand, the intention here is to try to use HRPC as a fence for policy makers that have a long-term interest in an area of internet development to be referred to the HRPC in order to address the human rights tradeoffs as a service that the IETF can provide. It would be a pity to reject that utility.

Reply:
Sorry I know these are all good intentions but HRPC is part of the IRTF and can't provide a service of the IETF. If that's the intention that is problematic and will also just lead to conflicts with the IETF rather than supporting or impact the IETF.

Reply:
I do not mean service in the formal sense.

Reply:
The sentence you are suggesting should be removed is not about IETF it's about external.

Just to move this forward I don't think we should merge this PR.

Reply:
Even in the informal sense, this is a service that an IRTF group cannot provide "for" the IETF. People who are interest to impact protocols to improve human rights, need to come to the IETF. People who are interested in research of how protocols impact human right, should go to HRPC.

Reply:
I think we need @csperkins to take a look at it.

Reply:
I agree that people who are interested to impact protocols to improve human rights, need to go to the IETF; people who are interested in understanding how protocols might impact human rights, or how human rights might impact protocols, should go to HRPC.

I also agree HRPC cannot "ensure" the standards process does anything. It can "support" the standards process by providing expertise to understand human right impacts and trade-offs, so those who are developing standards can make an informed choice in how they do so. And it can support policymakers by making them aware of the technical trade-offs and limitations, so they can make an informed choice when proposing policy. But its role is not to guide policymakers or standards developers to make a specific choice.

That expertise could be provided in the form of documents that address particular human rights issues and discuss ways in which different protocol design or policy choices can support or hinder those rights. That could be viewed as providing guidance, but provided such discussion is balanced and intended to inform rather than advocate, I'd argue that such documents are appropriate.

Whether this helps resolve the charter text, I'm not sure. Perhaps a key point is to highlight that the group is a venue for research, where standards developers and policymakers can develop a shared understanding of the issues, and not a venue where one comes to advocate for particular directions?

Reply:
I'm sorry, when I said service i should have said benefit. I think there's a focus on the wrong thing now. 

An example that is related to the sentence being deleted: when we are working on a document someone might bring up edits that would make the text more understandable to a policy or human rights audience. I think that suggestion should be considered in scope. 

It's really benign in that way.

Reply:
It doesn't matter if you call it service or benefit; this is nothing an IRTF group can or should provide "for" the IETF because it should happen in the IETF.

Maybe back to my original concern: The charter should not indicate that policy makers can come to this group present their concern and assume it will get fixed. While we might not be able to avoid that entirely, I think tis sentence increases the risk that people will interpret it this way.

Why do you think this sentence is needed? What's missing that you believe the group can't so if we remove it?

I think in this cases less words are better. That my approach for an easy solution but we can also try to find and alternative wording, however, I don't really understand what this sentences even provided.

Reply:
When we write documents we should consider the impacts outside of the ietf as documents will and should add to the larger debate on these issues. It's about making suggestions for external not internal reasons and having those be within scope. If we don't have this in the charter then external policy considerations could be said to be out of scope.
Maybe we can take an incisive approach and just not make a modification at this time.